src/app/components/media/MediaTimeline.js

```diff
@@ -0,0 +1,469 @@
+/* eslint-disable max-len */
+/* eslint-disable no-sequences */
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-return-assign */
+/* eslint-disable no-console */
+/* eslint-disable no-shadow */
```

Comment:
LOL

Reply:
I think Caio means: please don't disable lint check unless necessary :smile:

Reply:
I'll clean-up max-len, no-console, but I cannot live without no-shadow, a lot of destructuring in array operations shadows things

Reply:
@laurian The rest of us live without shadowing variables. To destructure without conflict, use `const { var1: uniqueName1, var2: uniqueName2 } = ...`.

Reply:
no-shadow, max-len, no-console fixed; the rest with the refactoring of that funky function (which BTW it intersects intervals for clips)

Reply:
fixed all